docs/examples/stable_diffusion/README.md

```diff
@@ -41,7 +42,8 @@ export USE_MEMORY_EFFICIENT_ATTENTION=1
 Use the NGC PyTorch container, to export and convert the models.
 
 ```
-docker run -it --gpus all -p 8888:8888 -v ${PWD}:/mount nvcr.io/nvidia/pytorch:22.08-py3
+# Replace yy.mm with year and month of release. Eg. 22.08
```

Comment:
You already mentioned this above. I don't think you need to repeat this.

Reply:
Fixed!